chains/335/0x0B608dE616aeC05df2da2BA3179f48e318060E2c/contract-information.json

```diff
@@ -1,4 +1,6 @@
 {
-  "address": "0x0B608dE616aeC05df2da2BA3179f48e318060E2c",
-  "name": "UniswapV2Factory"
-}
+    "address": "0x0B608dE616aeC05df2da2BA3179f48e318060E2c",
+    "name": "UniswapV2Factory",
+    "chainId": "335",
+    "description": " "
```

Comment:
Can we make these empty strings, and not strings with a single space in them?

Reply:
I think, in general, if we are going to enforce that there be a description we should enforce that the description has a minimum length